plugins/operators/dataset.py

```
@@ -9,9 +9,69 @@
 from dataclasses import asdict
 
 import fiftyone.operators as foo
+import fiftyone.operators.types as types
```

Ruff (0.15.21)

[warning] Use `from fiftyone.operators import types` in lieu of alias

Replace with `from fiftyone.operators import types`

(PLR0402)